### DISCUSSION TITLE
Changes in global warming

### DISCUSSION THREAD

**Discussion by sergejs-pogorelovs** [induktio/thinker#80] [Q&A]
Hi! I've been wondering how the mod changes how global warming works. 
I played a few games on the Librarian level and noticed that no sea level rise was occurring - even though the world was abundant with boreholes. I'd like the sea level rise to be as prevalent as in the vanilla.

I tried doing a search, found:
Details.md - "Ecological Damage. AI bases will not trigger sea level rise events unless the difficulty is Thinker or Transcend."
alphax.txt - "1, 2     ; Numerator/Denominator for frequency of global warming (1,2 would be "half" normal warming)."
I also found a bunch of changes that involve eco_damage_fix, but I think that those only increase the damage for the AIs, not lower it? Except a small change with the Tree farms.

Is this line in alphax.txt the only one that reduces the global warming? Or what Details.md is correct and code exists that I haven't found that reduces global warming for AIs on Thinker and Transcend levels?

**Comment by induktio** [comment]
Those are pretty much the changes included in the mod. There should be no other significant changes for ecological damage. Note that Tree Farms and similar always increasing clean minerals modifier has a significant effect if the production stays below eco damage threshold.
